democracy_club/templates/base.html

```diff
@@ -20,6 +20,8 @@
         {% block base_og_tags %}
             <meta property="fb:app_id" content="262795130596272" />
             <meta property="og:url" content="https://democracyclub.org.uk{{ request.path }}" />
+            <link rel="shortcut icon" href="{%  static "images/logo_icon.svg" %}">
+            <meta name="image" property="og:image" content="{{ CANONICAL_URL }}{% block og_image %}<img src="{% static "images/logo_icon.svg" %}" alt="" />{% endblock og_image %}
```

Comment:
This meta tag looks broken, there is an opening `<img` element in the content. Also I don't think we really need to add the `{% block og_image %}`. There is already the `base_og_tags` block if we wanted to override these elsewhere in the templates so don't think its necessary